src/nft/components/collection/Card.tsx

```diff
@@ -583,89 +652,95 @@ const ProfileNftDetails = ({ asset, hideDetails }: ProfileNftDetailsProps) => {
         </PrimaryDetails>
         {!hideDetails && <DetailsLink />}
       </PrimaryRow>
-      <Row justifyItems="flex-start">
-        <TruncatedTextRow
-          className={body}
-          style={{
-            color: themeVars.colors.textPrimary,
-          }}
-        >
-          {assetName()}
-        </TruncatedTextRow>
+      <ProfileAssetNameRow>
+        <TruncatedTextRow color="textPrimary">{assetName()}</TruncatedTextRow>
         {asset.susFlag && <Suspicious />}
-      </Row>
-      <TruncatedTextRow className={buttonTextMedium} style={{ color: themeVars.colors.textPrimary }}>
+      </ProfileAssetNameRow>
+      <FloorPriceRow>
         {shouldShowUserListedPrice && asset.floor_sell_order_price
           ? `${floorFormatter(asset.floor_sell_order_price)} ETH`
           : ' '}
-      </TruncatedTextRow>
-    </Box>
+      </FloorPriceRow>
+    </PrimaryRowContainer>
   )
 }
 
-const PrimaryRow = ({ children }: { children: ReactNode }) => (
-  <Row gap="8" justifyContent="space-between">
-    {children}
-  </Row>
-)
+const StyledPrimaryRow = styled(Row)`
+  gap: 8px;
+  justify-content: space-between;
+`
+
+const PrimaryRow = ({ children }: { children: ReactNode }) => <StyledPrimaryRow>{children}</StyledPrimaryRow>
+
+const StyledPrimaryDetails = styled(Row)`
+  justify-items: center;
+  overflow: hidden;
+  white-space: nowrap;
+`
 
 const PrimaryDetails = ({ children }: { children: ReactNode }) => (
-  <Row justifyItems="center" overflow="hidden" whiteSpace="nowrap">
-    {children}
-  </Row>
+  <StyledPrimaryDetails>{children}</StyledPrimaryDetails>
 )
 
+const PrimaryInfoContainer = styled.div`
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+`
+
 const PrimaryInfo = ({ children }: { children: ReactNode }) => {
-  return (
-    <Box overflow="hidden" whiteSpace="nowrap" textOverflow="ellipsis" className={body}>
-      {children}
-    </Box>
-  )
+  return <PrimaryInfoContainer>{children}</PrimaryInfoContainer>
 }
 
-const SecondaryRow = ({ children }: { children: ReactNode }) => (
-  <Row height="20" justifyContent="space-between" marginTop="6">
-    {children}
-  </Row>
-)
+const StyledSecondaryRow = styled(Row)`
+  height: 20px;
+  justify-content: space-between;
+  margin-top: 6px; ;
+`
+
+const SecondaryRow = ({ children }: { children: ReactNode }) => <StyledSecondaryRow>{children}</StyledSecondaryRow>
+
+const StyledSecondaryDetails = styled(Row)`
+  overflow: hidden;
+  white-space: nowrap;
+`
 
 const SecondaryDetails = ({ children }: { children: ReactNode }) => (
-  <Row overflow="hidden" whiteSpace="nowrap">
-    {children}
-  </Row>
+  <StyledSecondaryDetails>{children}</StyledSecondaryDetails>
 )
 
+const SecondaryInfoContainer = styled.div`
+  color: ${({ theme }) => theme.textPrimary};
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  line-height: 20px;
```

Comment:
The secondary line seems to have a different line height than what's currently in prod. Here's a gif comparing the two
![profileLineHeight](https://user-images.githubusercontent.com/11512321/221935416-88980bca-6937-4120-bb91-1ea24e62cd0a.gif)